code/game/turfs/open.dm

```diff
@@ -228,6 +231,8 @@
 				wet_time = max(0, wet_time-10)
 			if(T0C + 100 to INFINITY)
 				wet_time = 0
+	else if (GetTemperature() > BODYTEMP_COLD_DAMAGE_LIMIT)	//seems like a good place
+		MakeDry(TURF_WET_PERMAFROST)
```

Comment:
Could have a layer of water be on the tile after melting, the only way for ice to instantly evaporate is if you used plasma fire or something